framework/widgets/BaseListView.php

```diff
@@ -77,7 +77,8 @@
      */
     public $showOnEmpty = false;
     /**
-     * @var string the HTML content to be displayed when [[dataProvider]] does not have any data.
+     * @var string|bool the HTML content to be displayed when [[dataProvider]] does not have any data. When `false`,
+     * row with empty text will not be displayed.
```

Comment:
> row with empty text

These words confuse me. How about

<code>When \`false\`, table body will be empty.</code>